__tests__/app.test.js

```diff
@@ -286,3 +289,65 @@ describe("PATCH", () => {
     });
   });
 });
+
+describe("POST", () => {
+  describe("/api/articles/:article_id/comments", () => {
+    it("201: Add a new comments with all its values", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "Very good, very nice",
+      };
+      return request(app)
+        .post("/api/articles/9/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body: [comment] }) => {
+          expect(comment.comment_id).toBe(19);
+          expect(comment.body).toBe("Very good, very nice");
+          expect(comment.votes).toBe(0);
+          expect(comment.author).toBe("butter_bridge");
+          expect(comment.article_id).toBe(9);
+          expect(typeof comment.created_at).toBe("string");
+        });
+    });
+    it("400: should return error when passed an invalid request", () => {
```

Comment:
be a tad more descriptive in this description - what part is the invalid request?